streams.yml

```diff
@@ -54,10 +54,7 @@ rhel-9-golang-1.19:
 
 rhel-9-golang:
   image: openshift/golang-builder:v1.20.10-202310161945.el9.gbbb66ea
-  mirror: true
```

Comment:
Get images building before we disable the existing transform method. No need to change this file in this PR.